doc/api-docs/source/index.html.md

```diff
@@ -661,20 +730,20 @@ curl "https://dev.api.econia.exchange/account/0x1/open-orders"
 ```python
 import requests
 
-account_id = "0x1"
+account_address = "0x1"
 
 res = requests.get(
-    f"https://dev.api.econia.exchange/account/{account_id}/open-orders")
+    f"https://dev.api.econia.exchange/account/{account_address}/open-orders")
```

Comment:
This will get all open orders across all market accounts, correct?

E.g. if user 0xabc has one market account where they are the signing user, and another market account under the authority of custodian 234, this will get open orders for both market accounts?

Reply:
That is correct